src/test/server.ts

```diff
@@ -67,7 +67,7 @@ wss.on("connection", (ws) => {
       },
       getErrorMessage: (error: unknown) => "Something went wrong",
       getErrorCode: (error: unknown) => 100,
-    }).then((result) => ws.send(JSON.stringify(result)));
+    }).then((result) => ws.send(result));
```

Comment:
Those `JSON.stringify` calls in fetch and here actually confused me a bit. I basically expected fetch / the websocket to return raw data and `transcoder.deserialize` to convert it to something useable, however fetch is already internally parsing it.

Reply:
Currently, JSON serialization is expected to be done by the transport, not the transcoder. The use-case of transcoders is to support formats like [superjson](https://github.com/flightcontrolhq/superjson). I initially wanted an API like `{serializer: JSON}` (the default) or `{serializer: superjson}` as drop-in replacement, but this had some other drawbacks which I currently don't remember.
So the `JSON.stringify` part must be there, as otherwise the result will be an object rather than a string when using the default `identityTranscoder` (or superjson as in the other tests).